teuthology/orchestra/run.py

```diff
@@ -377,6 +377,7 @@ def run(
     label=None,
     timeout=None,
     cwd=None,
+    omit_sudo=False
```

Comment:
Please add a comment on why we need this since it's not actually used within the full framework right now.

Reply:
done, thanks